packages/core/src/Modules/Cashier/Components/payment-agent-details.jsx

```diff
@@ -5,13 +5,15 @@ import { Icon } from '@deriv/components';
 
 class Detail extends React.Component {
     render() {
-        const { action, icon, is_last_child, value, ...rest } = this.props;
+        const { action, icon, is_last_child, value, is_bold, ...rest } = this.props;
 
         return (
             <div className='payment-agent__accordion-content-line'>
                 <Icon icon={`Ic${icon}`} className='payment-agent__accordion-content-icon' color='secondary' />
                 <a
-                    className='payment-agent__contact cashier__paragraph'
+                    className={classNames('payment-agent__contact cashier__paragraph', {
+                        'payment-agent__contact--bold': is_bold,
```

Comment:
Why not wrap a `Text` component around the `value` child and set property to `bold`?

Reply:
Sure